app/models/state_file_base_intake.rb

```diff
@@ -64,19 +64,19 @@ def synchronize_filers_to_database
 
     if direct_file_json_data.primary_filer.present?
       attributes_to_update.merge!(
-        primary_first_name: direct_file_json_data.primary_first_name,
-        primary_middle_initial: direct_file_json_data.primary_middle_initial,
-        primary_last_name: direct_file_json_data.primary_last_name,
-        primary_birth_date: direct_file_json_data.primary_dob
+        primary_first_name: direct_file_json_data.primary_filer.first_name,
+        primary_middle_initial: direct_file_json_data.primary_filer.middle_initial,
+        primary_last_name: direct_file_json_data.primary_filer.last_name,
+        primary_birth_date: direct_file_json_data.primary_filer.dob
       )
     end
 
     if filing_status_mfj? && direct_file_json_data.spouse_filer.present?
       attributes_to_update.merge!(
-        spouse_first_name: direct_file_json_data.spouse_first_name,
-        spouse_middle_initial: direct_file_json_data.spouse_middle_initial,
-        spouse_last_name: direct_file_json_data.spouse_last_name,
-        spouse_birth_date: direct_file_json_data.spouse_dob
+        spouse_first_name: direct_file_json_data.spouse_filer.first_name,
+        spouse_middle_initial: direct_file_json_data.spouse_filer.middle_initial,
```

Comment:
I'm surprised to see this notation. I expected it to be a hash. How do we get to object? Am I overlooking something?

Reply:
If you're talking about the `spouse_filer.first_name` notation - that's a side effect of using the DfJsonAccessor (soon to be renamed) wrapper, as the `define_method` calls create a method "API" to the underlying data

Reply:
Ah, I see. I was thinking `direct_file_json_data` was the entry point to the json but it's actually an object with many entry points. Cool cool cool.